users/src/main/java/com/maksimpegov/users/exeption/ApiException.java

```diff
@@ -0,0 +1,15 @@
+package com.maksimpegov.users.exeption;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.time.ZonedDateTime;
+
+@Getter
+@AllArgsConstructor
+public class ApiException {
```

Comment:
Why you creating two exceptopns classes?

Reply:
ApiException is exception that my service will return, and ApiRuntimeException is exception that can be thrown in code, it extends RuntimeException(Java exception that cam be thrown at runtime)